docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -189,6 +189,22 @@ Folder that contains [provisioning]({{< relref "../../administration/provisionin
 
 `http`,`https`,`h2` or `socket`
 
+### min_tls_version
+
+Minimum TLS version that needs to be used in TLS Handshake. Possible Values are TLS1.2 and TLS1.3.
+If nothing is specified, TLS1.2 will be taken.
+
+### tls_ciphers
+
+Values accepted for TLS cipher. By default, this value is empty. Example value: "TLS_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256"
+For a complete list of supported ciphers and TLS versions, refer to: https://go.dev/src/crypto/tls/cipher_suites.go
```

Comment:
```suggestion
For a list of supported ciphers and TLS versions, refer to [Source file src/crypto/tls/cipher_suites.go](https://go.dev/src/crypto/tls/cipher_suites.go).
```

Reply:
This comment is fine. i will have made changes